### ISSUE TITLE
Screenshots flipped upside-down when using ddagrab in 1.1 Beta 10

### ISSUE THREAD

**Issue by jueank** [mjrgh/PinballY#252]
I just tested ddagrab in version 1.1 Beta 10, using the suggested image codec options: -frames:v 1 -update 1
I use exclusive fullscreen mode for VPX. Fullscreen optimisations are disabled (see here: https://devblogs.microsoft.com/directx/demystifying-full-screen-optimizations/)

The captured playfield was flipped upside down. Is this to be expected because you let users control the ffmpeg options for ddagrab completely on their own, or should this treated as a bug?

**Comment by mjrgh** [comment]
Did you include the [videoTransforms] in the source options per the help?

http://mjrnet.org/pinscape/downloads/PinballY/Help/CaptureFfmpegOptions.html#ddagrab-how-to

**Comment by jueank** [comment]
It's working now as intended with the entries from the help in 'Video Source Options'. Now that you mention it, it's obvious, but to be honest - I thought 'video source options' are really only relevant for video encoding, not for single images. But technically, it makes sense and I should have known if I had thought about it for 3 seconds longer...
You could change the name to 'Video & Image Source Options' if you want to?

**Comment by mjrgh** [comment]
> You could change the name to 'Video & Image Source Options' if you want to?

Good idea - I agree it's a little misleading that it only mentions video.

**Comment by jueank** [comment]
I would like to say thank you for including the ddagrab option! I have used it quite a lot in the last 2 days already. It works very reliably! Much better than gdigrab where I had to ALT-TAB out and ALT-TAB back into VPX to update the gdi framebuffer. It really is a time saving improvement!
